<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>InterfaceGPS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutRoot">
    <property name="spacing"><number>10</number></property>
    <property name="leftMargin"><number>12</number></property>
    <property name="topMargin"><number>12</number></property>
    <property name="rightMargin"><number>12</number></property>
    <property name="bottomMargin"><number>12</number></property>

    <item>
     <widget class="QFrame" name="topBarFrame">
      <property name="styleSheet">
       <string notr="true">QFrame{background:#171a21;border-radius:14px;} QLabel{color:white; font-size:16px;}</string>
      </property>
      <layout class="QHBoxLayout" name="topBarLayout">
       <property name="leftMargin"><number>12</number></property>
       <property name="topMargin"><number>10</number></property>
       <property name="rightMargin"><number>12</number></property>
       <property name="bottomMargin"><number>10</number></property>
       <property name="spacing"><number>16</number></property>

       <item><widget class="QLabel" name="lblSpeed"><property name="text"><string>0 km/h</string></property></widget></item>
       <item><widget class="QLabel" name="lblBattery"><property name="text"><string>Bat 100%</string></property></widget></item>
       <item><widget class="QLabel" name="lblGps"><property name="text"><string>GPS OK</string></property></widget></item>

       <item>
        <spacer name="spacerTop">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
         <property name="sizeHint" stdset="0"><size><width>200</width><height>20</height></size></property>
        </spacer>
       </item>

       <item>
        <widget class="QLabel" name="lblGear">
         <property name="text"><string>D</string></property>
         <property name="styleSheet"><string notr="true">QLabel{font-weight:800;font-size:20px;}</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QFrame" name="alertFrame">
      <property name="visible"><bool>false</bool></property>
      <property name="styleSheet">
       <string notr="true">QFrame{background:#6B4B16;border-radius:10px;} QLabel{color:white;}</string>
      </property>
      <layout class="QHBoxLayout" name="alertLayout">
       <property name="leftMargin"><number>10</number></property>
       <property name="topMargin"><number>8</number></property>
       <property name="rightMargin"><number>10</number></property>
       <property name="bottomMargin"><number>8</number></property>
       <property name="spacing"><number>10</number></property>

       <item>
        <widget class="QLabel" name="lblAlertTitle">
         <property name="text"><string>ALERTE</string></property>
         <property name="styleSheet"><string notr="true">QLabel{font-weight:800;}</string></property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAlertText">
         <property name="text"><string>...</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QStackedWidget" name="stackedPages"/>
    </item>

    <item>
     <widget class="QFrame" name="bottomNavFrame">
      <property name="styleSheet">
       <string notr="true">
QFrame{background:#171a21;border-radius:16px;}
QPushButton{height:54px;border-radius:12px;color:white;background:#222634;font-size:16px;}
QPushButton:pressed{background:#2d3245;}
       </string>
      </property>
      <layout class="QHBoxLayout" name="bottomNavLayout">
       <property name="leftMargin"><number>10</number></property>
       <property name="topMargin"><number>10</number></property>
       <property name="rightMargin"><number>10</number></property>
       <property name="bottomMargin"><number>10</number></property>
       <property name="spacing"><number>10</number></property>

       <item><widget class="QPushButton" name="btnHome"><property name="text"><string>Home</string></property></widget></item>
       <item><widget class="QPushButton" name="btnNav"><property name="text"><string>Nav</string></property></widget></item>
       <item><widget class="QPushButton" name="btnCam"><property name="text"><string>Cam</string></property></widget></item>
       <item><widget class="QPushButton" name="btnSettings"><property name="text"><string>Settings</string></property></widget></item>
      </layout>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
